### ISSUE TITLE
Build failure due to bad caching performed by maven-download-plugin

### ISSUE THREAD

**Issue by protoism** [gwt-plugins/gwt-eclipse-plugin#467]
This error has been reported in https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/465

I replicated the issue and probably found a solution in https://github.com/gwt-plugins/gwt-eclipse-plugin/pull/466

Here an Extract of "mvn -X generate-resources", which shows that maven-download-plugin is
picking gwt-2.8.2.zip instead of gwt-2.7.0.zip

```
Configuring mojo 'com.googlecode.maven-download-plugin:download-maven-plugin:1.2.1:
   (f) outputDirectory = /home/luca/gwt-eclipse-plugin/plugins/com.gwtplugins.gwt.ec
   (f) outputFileName = gwt-2.7.0.zip
   (f) overwrite = true
   (f) readTimeOut = 0
   (f) retries = 2
   (f) session = org.apache.maven.execution.MavenSession@5a4098d5
   (f) skip = false
   (f) skipCache = false
   (f) unpack = true
   (f) url = http://goo.gl/t7FQSn
 -- end configuration --
 Cache is: /home/luca/.m2/repository/.cache/maven-download-plugin
Got from cache: /home/luca/.m2/repository/.cache/maven-download-plugin/gwt-2.8.2.zip
```

**Comment by niloc132** [comment]
This can be closed now, by #466, correct?

**Comment by protoism** [comment]
Fixed by https://github.com/gwt-plugins/gwt-eclipse-plugin/pull/466
